src/jit/optcse.cpp

```diff
@@ -748,6 +765,102 @@ unsigned Compiler::optValnumCSE_Locate()
     return 1;
 }
 
+//------------------------------------------------------------------------
+// updateCseArrLenMap: Check if this compare is a tractable function of
+//                     an array length that is a CSE candidate, and insert
+//                     an entry in the cseArrLenMap if so.  This facilitates
+//                     subsequently updating the compare's value number if
+//                     the array length gets CSEd.
+//
+// Arguments:
+//    compare - The compare node to check
+
+void Compiler::updateCseArrLenMap(GenTreePtr compare)
```

Comment:
I named the field `cseArrLenMap`.  I suppose I could rename it to `optCseArrLenMap` and this method as you suggest, but I don't really understand what the benefit is... as a general rule, are we continuing this prefix naming stuff with new code?